app/README.md

```diff
@@ -19,12 +17,25 @@ We are proud to be using [Electron-React Boilerplate
 1. Install packages with `yarn`
 2. Run the application with `yarn start`
 
+## Contributing
+
+<!-- TODO: Move this to a CONTRIBUTING.md -->
+
 ### Code Style
 
--   Use `electron-log` for logging
--   Keep files small
+- Use `electron-log` for logging
+- Keep files small
+
+## Troubleshooting Local Development:
+
+#### DATABASE MALFORMED — TRIGGER ERROR
+
+If you see a console error that says the database is malformed, then you need to delete the databases the application reads from (so then it can create them from scratch again)
+
+1. `cd ~/.leftonread`
+2. `rm -r AddressBookFolder chat.db chat.db-shm chat.db-wal`
 
```

Comment:
@alexdanilowicz maybe wanna add the initial step you walked me through, which was giving the computer permission to read local files from vscode when developing?